src/mapbox/geolocation_control.js

```diff
@@ -20,7 +20,7 @@ class GeolocationControl extends GeolocateControl {
   }
 
   trigger() {
-    window.navigator.permissions.query({ name: 'geolocation' }).then(p => {
+    window.navigator.permissions && window.navigator.permissions.query({ name: 'geolocation' }).then(p => {
```

Comment:
can you revert this file & amend to avoid future conflict, it's an old artefact that is deleted in another branch.